community-call/20230831-slides.md

```diff
@@ -0,0 +1,272 @@
+
+---
+
+# Team IAM
+
+----
+
+# Team IAM
+## Most important item to share with the group that was discussed?
+
+Gap between Upstream/Testbed work and OSISM production status
+
+----
+
+# Team IAM
+## Any relevant news?
+
+First results from pentesting
+
+----
+
+# Team IAM
+## What was relevant for R5?
+
+Document the federation objectives, and the features already on the release.
+
+---
+
+# Team Ops
+
+> We build tooling and infrastructure design for easy, efficient and transparent ways to operate an SCS Cloud.
+
+----
+
+# Team Ops
+## Most important item to share with the group that was discussed?
+
+Work on the Observability for the Container Layer for Cloud Operators continues
+* Gonicus and dNation will discuss on how their Monitoring Stack can be used
+
+----
+
+# Team Ops
+## Any important decisions that are going to be taken?
+
+Status Page Test deployment will be done in GX-SCS
+
+----
+
+# Team Ops
+## What was relevant for R5?
+
+Most discussed items are relevant for R6, Status Page Frontend, Observability and the Central API
+
+---
+
+# Team Container
+
+----
+
+## Most important item to share with the group that was discussed?
+* R5 Release Notes :notebook:
+* Besides new features :gift: we'll put the newest versions of all components to the release
+
+
+----
+
+## Any important decisions that are going to be taken?
+* The E2E pipelines need to run on SCS Zuul
+
+----
+
+## What was relevant for R5?
+* Everything!
+
+----
+
+## Anything else important?
+* Issues with Gateway API deployment
+    * Cilium Pod in CrashLoopBackOff state
+    * Good chances to be fixed for R5
+
+
+---
+
+# SIG Community
+Minutes from the SIG: https://input.scs.community/2023-scs-sig-community
+![](https://input.scs.community/uploads/141063c5-8a41-4d8c-9c56-97282ef58f78.jpg)
+
+----
+
+# SIG Community
+
+## Most important item to share with the group that was discussed?
+* resuscitation of the "Friday Hacking Sessions" - we have always topics to work on together
+* team and SIG description on the docs page as well as our values and code of conduct
+
+----
+
+# SIG Community
+## Any important decisions that are going to be taken?
+* have "lessons learned" and technical blog posts also at the docs pages
+
+---
+
+# SIG Central API MVP
+
+----
+
+# SIG Central API MVP
+## Most important item to share with the group that was discussed?
+
+Discussed with Syself on details how Kubernetes cluster are created and how the Central API use the refrence implementation
+
+----
+
+# SIG Central API MVP
+## Any important decisions that are going to be taken?
+
+* To have a more detailed look on the Container Reference Implementation and continue the discussion on the next SIG Meeting
+* Want a label for GitHub issues related to the Central API
+* Use the Minihackathon Matrix Channel for Discussions
+
+---
+
+# Zuul for SCS
+
+- Very good overview given by dNation on e2e
+- Matrix-Room to coordinate efforts
+
+---
+
+# SIG Release
+
```

Comment:
@garloff do you have 1,2 sentences from the SIG Release?

Reply:
I was unable to insert that into the slides before the meeting.

Question is how we want to handle this now:
1. Should the slides stored here show exactly what was shown in the meeting, amended possibly by minutes from the minutes takes?
2. Or do we treat the slides also as minutes, and fill gaps in after the fact from things that were said in the meet, but not written on the slides?

For more formal meetings where decisions might be taken, I would always lean to no .1, but fot this one, no. 2 could be the most pragmatic approach.

I could then fill in those gaps ...

Reply:
I'd go with 2 - **however** we should keep it simple and don't spend hours on that (because that is what we have the minutes from the meetings themselves for).

Reply:
OK, this is what I did with https://github.com/SovereignCloudStack/minutes/pull/342/commits/5dc5ef655c65f7fa86c70dae50feeb9ce43268d8